WanderLost/WanderLost/Client/Components/Merchant.razor

```diff
@@ -0,0 +1,50 @@
+﻿<div class="merchant">
+    @if (SecondaryMerchant != null)
+    {
+        <div class="merchant__deck">
+            @if (TetriraryMerchant != null)
+            {
+                <Card ClassName="merchant__card merchant__card--tetriary" Merchant="@TetriraryMerchant" />
+            }
+            <Card ClassName="merchant__card merchant__card--secondary" Merchant="@SecondaryMerchant" />
+        </div>
+    }
+    @if (MainMerchant != null)
+    {
+        <Card ClassName="merchant__card" Merchant="@MainMerchant" />
+    }
+    else
+    {
+        <div>TODO</div>
+    }
+</div>
+
+@code {
+    [Parameter, EditorRequired]
+    public ActiveMerchantGroup MerchantGroup { get; set; } = default!;
+
+    public ActiveMerchant? MainMerchant
+    {
+        get
+        {
+            return MerchantGroup.ActiveMerchants.Count() > 0 ? MerchantGroup.ActiveMerchants.OrderByDescending(x => x.Votes).First() : null;
```

Comment:
It's generally simpler just to change this to 

MerchantGroup.ActiveMerchants.OrderByDescending(x => x.Votes).First**OrDefault**()

Then you don't have to check for an empty list, it will just return null if empty.